templates/ZtrBoardGame.Console/HelloService.cs

```diff
@@ -0,0 +1,46 @@
+using Microsoft.Extensions.Options;
+using Serilog;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using ZtrBoardGame.Configuration.Shared;
+
+namespace ZtrBoardGame.Console
+{
+    public class HelloService
+    {
+        private readonly NetworkSettings _networkSettings;
+        private readonly HttpClient _httpClient;
+
+        public HelloService(IOptions<NetworkSettings> networkSettings, HttpClient httpClient)
+        {
```

Comment:
use Microsoft.Ilogger injected trough constructor instead of those statics of serilog. Serilog is a backend which shouldn't leak to the app.

Reply:
@jules